### PR TITLE
Added file submission instructions and fixed typo

### PR DESCRIPTION
**Pull request by AxelBankuna**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by Sbonelo01 — APPROVED** [review]

File: content/sql/shop-project/_index.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ flavours:
 - none
 prerequisites:
   hard:
-  - topics/solo-learn/sql/4-challenges
-  - docker/intro-to-docker/
+    - topics/solo-learn/sql/4-challenges
```
Comment (ry-oc) [inline content/sql/shop-project/_index.md]: is the additional spaces here intentional?

File: content/sql/shop-project/_index.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ Save all of your instructions in a script file - you will submit this file on Gn
 
 ## Part 2: Querying a database
 
-Save all of your instructions in a script file - you will submit this file on Gnomio and github.
+Save all of your instructions in script files - you will submit these files on Gnomio and github.
```
Comment (ry-oc) [inline content/sql/shop-project/_index.md]: Please remove the reference to gnomio, it is no longer relevant.

**Review by ry-oc — CHANGES_REQUESTED** [review]

File: content/sql/shop-project/_index.md (hunk 1 above)
Comment (AxelBankuna) [reply content/sql/shop-project/_index.md]: It was in an attempt to shift the bullet points to the right but it had no affect and I forgot to undo that change.

File: content/projects/tdd/password-checker/part2/_index.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ tags:
 title: Add logging to password checker
 ---
 
-In part one of this exercise you created two functions. Now you are going to upgrade that program by adding some logging capabilities.
+In part one of this exercise you created two functions. Now you are going to upgrade that program by adding some logging capabilities using Log4J.
```
Comment (sheenarbw) [inline content/projects/tdd/password-checker/part2/_index.md]: Some people should be using the python logger. JS people use something else as well.

Please remember that this is nnot only used in the java course. 

You can add extra instructions. Eg: "If you are using java, please use log4J"

**Review by sheenarbw — CHANGES_REQUESTED** [review]
The instructions will mess with the js and python learners

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Same as before

**Review by EckardBerry — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**